lib/pid/include/PID.h

```diff
@@ -208,18 +208,20 @@ class PID {
   double compute(void);
 
   // Getters.
-  double getInMin();
-  double getInMax();
-  double getOutMin();
-  double getOutMax();
-  std::chrono::duration<double> getInterval();
-  double getPParam();
-  double getIParam();
-  double getDParam();
-  double getATunePParam();
-  double getATuneIParam();
-  double getATuneDParam();
-  double getSetPoint();
+  float getInMin() const;
```

Comment:
i think as a whole, we should be using doubles in fw. no reason not to.I don't think we even come close to hitting the limits of anything. shouldn't prematurely optimize.

Reply:
true, i guess here we should defintely return doubles bc the members are doubles

Reply:
Im suprised we didnt get a warning casting larger to type to smaller type

Reply:
apparently its allowed according to the standard: https://stackoverflow.com/questions/26091955/no-warning-when-demoting-double-to-float

Clang seems to have `-Wshorten-64-to-32` and `Wnarrowing`. We could make a note somewhere to add that when we switch to clang